public/locales/en/app.ftl

```diff
@@ -97,39 +100,35 @@ form-signup-checkbox = Get the latest from {-brand-Mozilla} and {-brand-name}.
 form-signup-submit = Sign up
 form-signup-error = Must be a valid email.
 
+
+found-breaches-headline = This could be a problem...
 no-breaches-headline = So far, so good
-no-breaches-blurb = Your account did not appear in our basic scan. That's good news, but data breaches can happen any time and there is still more you can do. Subscribe to {-product-name} for a full report, alerts when new breaches happen, and tips on protecting your passwords.
 
-found-breaches-headline = This might be a problem.
-found-breaches-singular-blurb = Your email address was compromised in the following breach.
-found-breaches-plural-blurb = Your email address was compromised in the following breaches.
+found-breaches-plural = Your email address appears in the following { $breachCount } breaches.
+found-breaches-singular = Your email address apppeared in the following breach.
+
+no-breaches-blurb = 
+  Your email address did not appear in our basic scan. 
+  That's good news, but data breaches can happen any time and there is still more you can do. 
+  Subscribe to Firefox Monitor for a full report, alerts when new breaches happen, and tips on protecting your passwords.
+
+featured-breach-results = 
```

Comment:
@stasm I'm still running into some selector trouble here in "featured-breach-results". As is, this throws a "ctor is not a constructor" error. If I change [one] to [1], the string will return correctly only if the value of $breachCount is exactly 1. I'm totally puzzled as I recreated this over in the Fluent playground where it appears to be working exactly as expected. Have you run into anything like this?

Reply:
This might be related to the `Intl.PluralRules` API. It's still a fairly new feature in JavaScript. Looking at the [browser support matrix](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/PluralRules#Browser_compatibility), is there a chance you're testing on Safari? In order to support Safari as well as older browsers, you'll need to polyfill `Intl.PluralRules`. I recommend [`intl-pluralrules`](https://www.npmjs.com/package/intl-pluralrules).

Background: Fluent relies on this API to match variant keys which are not numbers. When you use `[one]` Fluent maps the numerical value of `$breachCount` to a plural category. Categories are [defined in CLDR](http://www.unicode.org/cldr/charts/29/supplemental/language_plural_rules.html). For English, they're `one` and `other`. Other languages have fewer or more.

Reply:
Aha, node.js is probably the culprit here - it doesn't have support for `Intl.PluralRules` API. I'll see if I can add that to our `package.json` and get this to work.